app/main.py

```diff
@@ -1,3 +1,46 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        config = json.load(file)
+
+    customers = []
+    for cas in config["customers"]:
+        customers.append(Customer(
+            name=cas["name"],
+            product_cart=cas["product_cart"],
+            location=tuple(cas["location"]),
+            money=cas["money"],
+            car=cas["car"]
+        ))
+    shops = []
+    for sh in config["shops"]:
+        shops.append(Shop(name=sh["name"],
+                          location=sh["location"], products=sh["products"]))
```

Comment:
use list comprehension here 
and use unpacking  
```suggestion
    shops = []
    for sh in config["shops"]:
        shops.append(Shop(**sh))
```